course_info/icommons.py

```diff
@@ -92,6 +108,31 @@ def _get_schools(self, school_id=None, **kwargs):
             raise ICommonsApiValidationError(str(e))
         return rv
 
+    def _course_info_instructor_list(self, course_instance_id=None, **kwargs):
+
+        # use kwargs to append 'people' to the resource url and to get course people data
+        kwargs['append_to_path'] = 'people'
+
+        instructors = None
+        rv = self._get_resource('course_instances', course_instance_id, **kwargs)
```

Comment:
Yes, that would break. At one point I think I had canvas course id  as a param too,  but moved it upstream. Removing default.